src/v/model/transform.cc

```diff
@@ -74,6 +74,16 @@ void append_vint_to_iobuf(iobuf& b, int64_t v) {
 
```

Comment:
In the commit message for fc899ae960b623f2a61b54a9022019ec36fb2d8c:

Enclosed delta should always be greater than _**or equal to**_ zero.

Reply:
done